src/views/Account/Rewards/PromotionSummary.tsx

```diff
@@ -0,0 +1,230 @@
+import FeatherIcon from 'feather-icons-react'
+import Link from 'next/link'
+import classNames from 'classnames'
+import { BigNumber } from 'ethers'
+import { formatUnits } from 'ethers/lib/utils'
+import { useTranslation } from 'react-i18next'
+import { getNetworkNiceNameByChainId, sToMs, numberWithCommas } from '@pooltogether/utilities'
+import { useToken } from '@pooltogether/hooks'
+import { format } from 'date-fns'
+import { TokenIcon } from '@pooltogether/react-components'
+import { Trans } from 'react-i18next'
+
+import { CHAIN_ID, SECONDS_PER_DAY } from '@constants/misc'
+// import { SummaryWell } from './SummaryWell'
+// import { TokenDisplay } from './TokenDisplay'
+
+interface PromotionSummaryProps {
+  chainId: number
+  token: string
+  startTimestamp: number
+  tokensPerEpoch: BigNumber
+  epochDuration: number
+  numberOfEpochs: number
+  networkName: string
+  className?: string
+}
+
+const days = (numberOfEpochs, epochDuration) => {
```

Comment:
Can we rename this to something more descriptive - `getPromotionDurationInDays` or something like that